src/components/AddFriend/AddFriend.tsx

```diff
@@ -1,65 +1,46 @@
 import React, { useState } from "react";
 import {
   TextField,
+  Button,
   Box,
   Snackbar,
   IconButton,
   InputAdornment,
 } from "@mui/material";
 import SendIcon from "@mui/icons-material/Send";
-import { checkFriendRequestStatus, sendFriendRequest } from "@/services/friendService";
+import { sendFriendRequest } from "@/services/friendService";
 import { getUserByEmail } from "@/services/userService";
 import { useUser } from "@/contexts/UserContext";
 
 const AddFriend: React.FC = () => {
   const [emailInput, setEmailInput] = useState("");
   const [snackbarOpen, setSnackbarOpen] = useState(false);
-  const [error, setError] = useState("");
+  const [error, setError] = useState(false);
   const [helperText, setHelperText] = useState("");
-  const { user } = useUser(); // Assuming useUser returns an object with a user field
```

Comment:
this gets user.user from the user context. bring this back so we dont have to write user.user everywhere. {} destructures the object.